src/drive/mobile/lib/cozy-helper.js

```diff
@@ -29,6 +29,7 @@ export const initClient = url => {
         scopes: [
           'io.cozy.files',
           'io.cozy.contacts',
+          'io.cozy.apps',
```

Comment:
Maybe `io.cozy.apps:GET`, that's what the manifest has, and we really don't want permissions to write on that doctype^^

Reply:
Fixed